.github/workflows/publish.yml

```diff
@@ -35,9 +35,10 @@ jobs:
       environment: ${{ github.event.inputs.environment }}
       docs-only: ${{ fromJSON(github.event.inputs.docs-only) }}
       golangci-lint-version: '2.1.6'
-      playwright-secrets: |
-        AWS_ACCESS_KEY_ID=e2e:accessKey
-        AWS_SECRET_ACCESS_KEY=e2e:secretKey
+      run-playwright: false
+      # playwright-secrets: |
+      #   AWS_ACCESS_KEY_ID=e2e:accessKey
+      #   AWS_SECRET_ACCESS_KEY=e2e:secretKey
```

Comment:
There's no need to disable Playwright anymore :smiley:

Reply:
🤦🏻‍♀️